packages/bundler/src/modules/ReputationManager.ts

```diff
@@ -178,7 +178,8 @@ export class ReputationManager {
       this.entries[rep.address] = {
         address: rep.address,
         opsSeen: rep.opsSeen,
-        opsIncluded: rep.opsIncluded
+        opsIncluded: rep.opsIncluded,
+        status: rep.status
```

Comment:
why do you need the to save the reputation status?
if you look at the [getStatus function](https://github.com/eth-infinitism/bundler/blob/c31079a34f68fafd46edafe034bcb1893aaf3bb4/packages/bundler/src/modules/ReputationManager.ts#L128), it doesn't read this value, but always calculate it based on opsSeen/opsIncluded